.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
 python:
+  - 3.4
```

Comment:
do we want to test both on all commits? or should we just do this on dev, master (and pr's to dev, master)?